src/client/MenuWindow.cpp

```diff
@@ -2,186 +2,372 @@
 #include <iostream>
 #include <json/json.h>
 #include <fstream>
+#include <string>
+#include <codecvt>
 
-#define WINDOW_LENGTH 1600
-#define WINDOW_WIDTH 900
+#define REFRESH_TIME 30
 
-#define TITLE_SIZE 200
+#define BUTTON_CREAT 1
+#define BUTTON_USERNAME 2
+#define BUTTON_SERVER 3
+#define BUTTON_PORT 4
+#define BUTTON_LOAD 4
+#define BUTTON_ID 5
+#define BUTTON_PLAYER 5
+#define BUTTON_SEED 6
+#define BUTTON_CONNECT 6
+#define BUTTON_START 7
+
+#define CREATE_GAME "new"
+
+#define ASCI_BEGIN 41
+#define ASCI_END 123
 
 #ifndef RESOURCES_PATH
 #define RESOURCES_PATH "../resources"
 #endif
 
+const sf::Color buttonColor = sf::Color(247, 200, 195, 255);
+
 namespace client
 {
 
-    /*!
-     * @brief Constructor
-     *
-     * Constructor of MenuWindow class
-     */
-    MenuWindow::MenuWindow()
-    {
-        loadMenuTexture();
-    }
+/*!
+ * \brief Constructor
+ *
+ * Constructor of MenuWindow class
+ */
+MenuWindow::MenuWindow()
+{
+    currentMenu = &menuButtons;
+    currentText = &menuTexts;
+}
 
-    /*!
-     * @brief Display all the menu on the screen
-     */
-    void MenuWindow::displayWindow()
-    {
+/*!
+ * \brief Display all the menu on the screen
```

Comment:
``
```suggestion
 * @brief Display all the menu on the screen
```
Same for all /brief